README.rst

```diff
@@ -401,9 +401,9 @@ We use the official `elastic app search python client <https://github.com/elasti
 
```

Comment:
The docs have a link above this to app-search-python - which is the deprecated library - could you update it so it points to https://github.com/elastic/enterprise-search-python?

Reply:
Done

Reply:
Theres the second link that still points to the old repo.

Reply:
It should be ok now

Reply:
thanks!